react/orders101/src/features/navbar/Navbar.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext, useState } from "react";
+import BasketOpen from "../BasketOpen";
+import Hamburger from "../Hamburger";
+import { ShoppingCartContext } from "../../App";
+import Cart from "../cart/Cart";
+
+function Navbar() {
+  let [isCartOpen, setIsCartOpen] = useState(false);
+
+  const closeCart = () => {
+    setIsCartOpen(false);
+  };
+  const openCart = () => {
+    setIsCartOpen(true);
+    console.log(isCartOpen);
```

Comment:
I know this is just for debugging purposes but you'd probably get some linting errors from these lines.

Reply:
console log removed